src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -425,6 +425,7 @@ public function load(array $configs, ContainerBuilder $container)
         $this->registerAnnotationsConfiguration($config['annotations'], $container, $loader);
         $this->registerPropertyAccessConfiguration($config['property_access'], $container, $loader);
         $this->registerSecretsConfiguration($config['secrets'], $container, $loader);
+        $loader->load('config.php');
```

Comment:
I think we should merge this file with another one, eg services.php, that'b be easier to grasp, especially for one service

Reply:
Sure!